### ISSUE TITLE
Focus for file-links in file-upload table is invisible

### ISSUE THREAD

**Issue by Magnusrm** [Altinn/app-frontend-react#3997]
### Description of the bug

When tabbing to a download link for a file in a file-upload table (either with or without tags) the focus state is not visible.

Additionaly the table itself can receive focus but is not an accessible element. This is misleading and should not be focusable.

### Steps To Reproduce

1. got to a form page with a file-upload component
2. upload an attachment
3. place focus using TAB on the donwload link
4. see that the focus state is not visible
